src/framework/standard/help_commands.rs

```diff
@@ -819,14 +822,14 @@ fn flatten_group_to_string(
         .join(&format!("\n{}", &repeated_indent_str));
 
 
-    if group.command_names.len() >= 1 {
+    if group.command_names.is_empty() {
```

Comment:
this looks wrong

Reply:
That's what I get for copy and pasting. Good catch!